docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -23,8 +23,7 @@ embedded assets. This can be different for older releases.
 {{< toc >}}
 
 ## Download
-
```

Comment:
Why is the new line between heading and text only removed in this file?

Reply:
I think it looks tidier when editing (although it makes no difference when rendered).
Shall I add them back?

Reply:
I think markdown rule MD022 suggests that "Headers should be surrounded by blank lines"

Reply:
yes the blank line is required.